lib/api/formtools/actions.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+/**
+ * Parse some action disabled properties.
+ * @param  {Object} record Mongoose model record.
+ * @param  {Array|Object} actions Actions to parse.
+ * @return {Array} Parsed actions.
+ */
+const parseDisabledProperties = (record, actions) => new Promise((resolve, reject) => {
+
+    const parsedActions = [].concat(actions);
+    const actionsToBeParsed = [];
+
+    parsedActions.forEach((action) => {
```

Comment:
I feel it would be cleaner and easier to read by using `parsedActions.filter` first to filter based on `action.disabled`. It could remove that `if` statement below.